dags/openshift_nightlies/scripts/run_benchmark.sh

```diff
@@ -33,5 +33,9 @@ setup
 cd /home/airflow/workspace
 ls
 cd e2e-benchmarking/workloads/$workload
+
+if [[ $workload == "network-perf" ]]; then
```

Comment:
It will be better to run the python library install command in the e2e-benchmarking repo itself.

Reply:
Yeah I looked at the actual code in network-perf and it looks like it *should* do a pip install everytime the script runs. I saw that it wanted me to do that in the readme which is why I added it but that may just be outdated so I'll remove this and see if the uperf workloads still work properly.

Reply:
yes, else we can add the pip install command in https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/network-perf/common.sh